front-end/src/app/shared/services/report.service.ts

```diff
@@ -90,6 +90,6 @@ export class ReportService implements TableListService<Report> {
   }
 
   public startAmendment(report: Report): Observable<string> {
-    return this.apiService.post(`${this.apiEndpoint}/${report.id}/amend/`, {});
+    return this.apiService.post(`${this.apiEndpoint}/${report.apiToken}/${report.id}/amend/`, {});
```

Comment:
`apiToken` is a bit misleading about what this does.  Maybe `apiPath` or `apiPrefix`?

I think you can avoid all of this routing with my comment in the api pr though.  Just keep this url what it was before and have the api handle the amending
https://github.com/fecgov/fecfile-web-api/pull/666#pullrequestreview-1818806364